# HG changeset patch
# User Carnë Draug <carandraug+dev@gmail.com>
# Date 1320023141 0
#      Mon Oct 31 01:05:41 2011 +0000
# Node ID 6d59b141e65d17ff9f692890c34d47e4b87e5e27
# Parent  55ab001af1b8032a2fdc73ee663a140f626dcd55
Add python to manual and respect format rules of copyright block

diff --git a/doc/interpreter/system.txi b/doc/interpreter/system.txi
--- a/doc/interpreter/system.txi
+++ b/doc/interpreter/system.txi
@@ -297,16 +297,18 @@ higher-level functions.
 @DOCSTRING(system)
 
 @DOCSTRING(unix)
 
 @DOCSTRING(dos)
 
 @DOCSTRING(perl)
 
+@DOCSTRING(python)
+
 @DOCSTRING(popen)
 
 @DOCSTRING(pclose)
 
 @DOCSTRING(popen2)
 
 @DOCSTRING(EXEC_PATH)
 
diff --git a/scripts/miscellaneous/python.m b/scripts/miscellaneous/python.m
--- a/scripts/miscellaneous/python.m
+++ b/scripts/miscellaneous/python.m
@@ -1,10 +1,10 @@
 ## Copyright (C) 2008-2011 Julian Schnidder
-## Copyright (C) 2011 Carnë Draug <carandraug+dev@gmail.com>
+## Copyright (C) 2011 Carnë Draug
 ##
 ## This file is part of Octave.
 ##
 ## Octave is free software; you can redistribute it and/or modify it
 ## under the terms of the GNU General Public License as published by
 ## the Free Software Foundation; either version 3 of the License, or (at
 ## your option) any later version.
 ##
@@ -22,16 +22,18 @@
 ## @deftypefnx {Function File} {[@var{output}, @var{status}] =} python (@var{scriptfile}, @var{argument1}, @var{argument2}, @dots{})
 ## Invoke python script @var{scriptfile} with possibly a list of
 ## command line arguments.
 ## Returns output in @var{output} and status
 ## in @var{status}.
 ## @seealso{system}
 ## @end deftypefn
 
+## Author: Carnë Draug <carandraug+dev@gmail.com>
+
 function [output, status] = python (scriptfile = "-c ''", varargin)
 
   ## VARARGIN is intialized to {}(1x0) if no additional arguments are
   ## supplied, so there is no need to check for it, or provide an
   ## initial value in the argument list of the function definition.
 
   if (ischar (scriptfile)
       && ((nargin != 1 && iscellstr (varargin))
